src/scripts/main.js

```diff
@@ -1,3 +1,28 @@
 'use strict';
 
-// write code here
+const people = document.querySelector('ul');
+const persons = [...people.querySelectorAll('li')];
+
+function toNumber(salary) {
+  return +salary.replace(/[^\d]/g, '');
+}
+
+function sortList(list) {
+  list
+    .sort((a, b) => toNumber(b.dataset.salary)
+    - toNumber(a.dataset.salary));
+
+  people.append(...list);
+}
+
+function getEmployees(list) {
+  return [...list].map(person => {
```

Comment:
it has no sense just
```suggestion
  return list.map(person => {
```